RELEASE.md

```diff
@@ -8,32 +8,32 @@ This is a document to describe the release process for the slsa-github-generator
 
 <!-- toc -->
 
-- [Prerequisites](#prerequisites)
-- [Release candidate](#release-candidate)
-  - [Update CHANGELOG](#update-changelog)
-  - [RC tagging](#rc-tagging)
-  - [Verify RC version references](#verify-rc-version-references)
-  - [Adversarial verifier tests](#adversarial-verifier-tests)
-    - [Go builder verifier test](#go-builder-verifier-test)
-    - [Generic generator verifier test](#generic-generator-verifier-test)
-    - [Container generator verifier test](#container-generator-verifier-test)
-  - [Adversarial builder tests](#adversarial-builder-tests)
-    - [Adversarial Go builder](#adversarial-go-builder)
-    - [Adversarial generic generator](#adversarial-generic-generator)
-    - [Adversarial container generator](#adversarial-container-generator)
-  - [Finalize release candidate.](#finalize-release-candidate)
-  - [Code Freeze](#code-freeze)
-- [Finalize release](#finalize-release)
-  - [Update CHANGELOG](#update-changelog-1)
-  - [Release tagging](#release-tagging)
-  - [Verify final version references](#verify-final-version-references)
-  - [Final adversarial tests](#final-adversarial-tests)
-  - [Reference Actions at main](#reference-actions-at-main)
-  - [Update documentation](#update-documentation)
-  - [Update verifier](#update-verifier)
-  - [Finish the release.](#finish-the-release)
-  - [Update the starter workflows](#update-the-starter-workflows)
-  - [Announce](#announce)
+- [How to cut a release](#how-to-cut-a-release)
```

Comment:
nit: I don't think the TOC needs to include the doc title.

Reply:
Fixed. I have a Markdown linter that auto-updates the TOC, missed this.